src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 10;
+    private static final int BOUND_OF_CASES = 10;
+    private static final int DEFAULT_RADIUS = 12;
+    private static final int FIGURE_COUNT = 3;
+    private static final int ORIGIN = 1;
+    private final Figure[] randomFigures = new Figure[3];
+    private final Figure[] defaultFigures = new Figure[3];
```

Comment:
The array size should not be hardcoded. Use FIGURE_COUNT to define the size of the defaultFigures array to maintain consistency and avoid magic numbers.